src/lib/containers/SynapseTableCardView.tsx

```diff
@@ -97,27 +102,60 @@ class SynapseTableCardView extends React.Component<SynapseTableCardViewProps, Sy
         }
     }
 
+    componentDidMount() {
+        // we try to load one page of data ahead of cards, this allows the "view more" behavior 
+        // to be instant
+        this.getBufferData()
+    }
+
+    getBufferData() {
+        // Load data ahead of the currently displayed data, do this recursively in case it needs more time
+        if (!this.state.hasLoadedBufferData) {
+            setTimeout(() => {
+                if (!this.props.getLastQueryRequest) {
+                    // parent component still setting up
+                    this.getBufferData()
+                    return
+                }
+                let queryRequest = this.props.getLastQueryRequest!();
+                if (!queryRequest.query) {
+                    // parent component still setting up
+                    this.getBufferData()
+                    return
+                }
+                let offset = queryRequest.query.offset!;
+                // if its a "previous" click subtract from the offset
+                // otherwise its next and we paginate forward
+                offset += 25;
```

Comment:
pull out as constant.  have you tested with 3 pages?

Reply:
have not, can test that now